sdk/clients/request.go

```diff
@@ -16,6 +17,7 @@ var httpClient = &http.Client{
 	Transport: &http.Transport{
 		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
 	},
+	Timeout: time.Duration(5 * time.Second),
```

Comment:
magic number?

Reply:
well, NetClientTimeout provides ability to change 5sec defaults